beginner-guides.md

```diff
@@ -0,0 +1,65 @@
+# Beginner guides
+
+These guides are aimed at beginners who want to submit their firt pull requests or that want to start contributing to open source.
+
+This is a step-by-step guide that may help guide you on where to start if you want to contribute to any of our projects listed in <https://github.com/survival>, but should also be helpful if you wanted to contribute to any open source project that lives on GitHub.
+
+
+## What you will need
+
+* You need to know how to use git, either through a GUI applicaton or through the terminal. CodeSchool has [a good tutorial on git](https://try.github.io/levels/1/challenges/1).
+
+* Since most of our projects are in [GitHub](https://github.com), you will need to have an account there. Jessica Lord has [the best tutorial on Github](https://github.com/jlord/git-it-electron).
+
+* It helps to know how to use the terminal as well, since most of our projects need you to run commads, for example, to run the tests. Codebar has a starter tutorial on [the command line](http://tutorials.codebar.io/command-line/introduction/tutorial.html).
+
+
+## Where to start
+
+* Search for issues in any of our repos with the label `good-first-issue`.
+
+* When you find a project you want to contribute to, fork the repository by clicking the "Fork" button on the top right of the project's home page.
+
+![alt text][forkit]
+
+* This will create a copy of our project in your GitHub account. This is because, in order to push to a repository, you need to have writing permissions, and only the owners of a repo have them. But even if you don't have writing permissions to our project, you will have writing permissions to your copy of the project, and you can push code there and then send it to us (will tell you how to do this below).
+
```

Comment:
This will create a copy of our project in your GitHub account. This allows you to modify the project safely in your account and once you're happy with the changes you can submit it to us (see **Starting your contribution** below)